android/java/org/chromium/chrome/browser/ntp/BraveNewTabPageLayout.java

```diff
@@ -644,11 +681,18 @@ protected void onDetachedFromWindow() {
                 BraveActivity.getBraveActivity().setNewsItemsFeedCards(mNewsItemsFeedCard);
```

Comment:
there are no checks on `if (BraveActivity.getBraveActivity() != null) {` but I see them in other places. Should we check it here as well?